mesheryctl/internal/cli/root/experimental/experimental.go

```diff
@@ -45,6 +44,6 @@ var ExpCmd = &cobra.Command{
 }
 
 func init() {
-	availableSubcommands = []*cobra.Command{mesh.MeshCmd, filter.FilterCmd}
+	availableSubcommands = []*cobra.Command{filter.FilterCmd}
```

Comment:
Good

Reply:
@alphaX86 
I saw your comment in issues
> the filter is still under exp and we need to bring out of that

Here you removed `mesh`  cmd. any specific reason for this

Reply:
`mesh` cmd isn't supposed to be experimental. It is already live via `mesheryctl mesh`. Since it wasn't overlooked for long time, some may misunderstand that `mesh` is under `exp` but it really isn't. So had to remove from the list. I hope its now clear @Meyazhagan